src/test/java/com/google/devtools/build/lib/skylark/SkylarkIntegrationTest.java

```diff
@@ -787,7 +787,8 @@ public void testRuleClassImplicitOutputs() throws Exception {
     scratch.file(
         "test/skylark/extension.bzl",
         "def custom_rule_impl(ctx):",
-        "  files = [ctx.outputs.lbl, ctx.outputs.list, ctx.outputs.str]",
+        "  files = [ctx.outputs.lbl, ctx.outputs.str]",
+        "  files += ctx.outputs.list",
```

Comment:
This means that your change is a breaking change. We try to avoid breaking changes. If we have to do it, we have to make sure we're not breaking anyone.